sonar-db/src/main/resources/org/sonar/db/ce/CeActivityMapper.xml

```diff
@@ -18,12 +18,26 @@
     ca.updated_at as updatedAt,
     ca.is_last as isLast,
     ca.is_last_key as isLastKey,
-    ca.execution_time_ms as executionTimeMs
+    ca.execution_time_ms as executionTimeMs,
+    ca.error_message as errorMessage,
+    <choose>
```

Comment:
Is it really mandatory to return the field hasErrorStacktrace in SQL ? 
Why not moving this code in the DTO, for instance : 
```
public boolean isHasErrorStacktrace() {
   return errorStacktrace != null;
}
```